internal/parser/issue29_test.go

```diff
@@ -0,0 +1,31 @@
+package parser
+
+import "testing"
+
+func TestIssue29(t *testing.T)  {
+	NegativeTest{
+		Name:  "issue29",
+		Query: "CREATE TABLE(n,FOREIGN KEY()REFERENCES n ON DELETE CASCADE)",
+	}.Run(t)
+}
+
+func TestIssue29WithoutTableName(t *testing.T) {
+	NegativeTest{
+		Name:  "issue29",
```

Comment:
You are already giving a better name in the method name, please use it here as well
```suggestion
		Name:  "issue29 without table name",
```